### ISSUE TITLE
RankedLinearSVC probabilities

### ISSUE THREAD

**Issue by cdbethune** [uncharted-distil/distil-primitives#183]
Right now in the binary label case the RankedLinearSVC primitive we will generate a set of ranks based on the decision function and return that as its output along with the labels.  This ranking makes sense in the binary case, but doesn't necessarily make sense when we're dealing with multiple labels.  
1. Add a hyperparameter to control whether or not ranking should be generated instead of actual confidences.  This is only valid in the case where the label is binary.
1. Generate confidences in the case where the label is binary and ranks aren't requested, or in the case where the label is not binary.  A discussion of how to generate confidences from labels can be found here (there may be other ways):

https://erogol.com/predict-probabilities-sklearn-linearsvc/

I'm not sure how much overhead this will impose.
